pkg/rtc/room.go

```diff
@@ -1069,6 +1069,15 @@ func (r *Room) pushAndDequeueUpdates(pi *livekit.ParticipantInfo, isImmediate bo
 				return nil
 			}
 		}
+	} else {
+		ep := r.GetParticipant(identity)
```

Comment:
This grabs the room lock (read lock). But, `batchedUpdatesMu` is very tightly scoped and did not see a chance of a lock order reversal/deadlock, but please let me know if I am missing something.